nncf/config.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, *args, **kwargs):
     def from_json(cls, path) -> 'NNCFConfig':
         with open(path) as f:
             loaded_json = json.load(f)
-        NNCFConfig.validate(loaded_json)
+        # NNCFConfig.validate(loaded_json)
```

Comment:
This should be uncommented. Please add all new parameters with description to the `config_schema`.

Reply:
@ljaljushkin Need help to jointly decide how and what to expose and sort out the json validation.

Reply:
uncommented and created first schema for autoq. Pending revision on description on key that applies to hawq and autoq